pkg/account/import.go

```diff
@@ -88,7 +90,15 @@ func ImportKeyStore(keyPath, name, passphrase string) (string, error) {
 	} else if store.DoesNamedAccountExist(name) {
 		return "", fmt.Errorf("account %s already exists", name)
 	}
-
+	key, err := keystore.DecryptKey(keyJSON, passphrase)
+	if err != nil {
+		return "", err
+	}
+	b32 := address.ToBech32(key.Address)
+	hasAddress := store.FromAddress(b32) != nil
+	if hasAddress {
+		return "", fmt.Errorf("address %s already exists in keystore", b32)
+	}
 	ks := store.FromAccountName(name)
 	_, err = ks.Import(keyJSON, passphrase, passphrase)
```

Comment:
Import decrypts the keysJSON, which takes some time. Since we already know that the keystore file is valid at this point, we can just copy over the file to the correct directory, this will be faster.

Reply:
I've tested ok the code after adding:

``` Go
uDir, _ := homedir.Dir()
path := filepath.Join(uDir, c.DefaultConfigDirName, c.DefaultConfigAccountAliasesDirName, name)	
err = os.MkdirAll(path, 0600)
if err != nil {
	return "", err
}
err = ioutil.WriteFile(filepath.Join(path, filepath.Base(keyPath)), keyJSON, 0600)
if err != nil {
	return "", err
}
```

just wondered if I should leave that in
- the import.go 
- another package like common
- a private function of account package